src/odemis/gui/plugin/__init__.py

```diff
@@ -559,6 +559,22 @@ def on_close(self, btnid, _):
         self.streambar_controller.clear()
         self.EndModal(btnid)
 
+    def _pause_settings(self):
+        """ Pause the settings of the GUI and save the values for restoring them later """
```

Comment:
If it's supposed to be called by external code, don't make the method "private". => `pauseSettings`/`resumeSettings`.

Reply:
Also, the docstring can be improved. In particular, it does _not_ save the values (ie, if they are changed, on resume the values will be displayed changed). I'd suggest something like:
`Pause the settings widgets. They will be disabled and the value frozen even when the VAs are changed`.